Cargo.toml

```diff
@@ -21,10 +21,24 @@ status = "actively-developed"
 
 [dependencies]
 anyhow = { workspace = true }
-clap = { workspace = true, features = ["color", "derive", "error-context", "help", "std", "suggestions", "usage"] }
+clap = { workspace = true, features = [
```

Comment:
These changes seem out of scope for the PR

Reply:
This is applied by my toml formatter, so it is likely best practice.

Reply:
Oh and generally a lot of us other maintainers followers of what I call the "campfire" or "campground" rule which is "leave it cleaner than you found it." This was mostly unintentional because it was automatic, but I don't think it is a problem to leave it cleaner than it was

Reply:
This change makes it very difficult to lexicographically sort the imports and copy-paste this line

Reply:
We can change back later if it becomes a thorn in our sides

Reply:
Sure, but do note that had you not made this change, there would be no merge conflict with #432 - this is primarily why I'm against out-of-scope changes in PRs and prefer PRs to be focused on doing one thing at a time - it lets us move faster with less friction